hw2/03-count.js

```diff
@@ -1,3 +1,19 @@
 // Add your code here
 
+const input = document.getElementById('userInput');
+const textField = document.getElementById('text');
+const highlightedText = document.getElementById('highlightedText');
+
+const handleKeyDown = function handleUserKeyDown(event) {
+  if (event.keyCode === 13) {
+    const searchTerm = input.value.trim();
+    const searchRegex = new RegExp(`\\b${searchTerm}\\b`, 'gi');
+    const searchContent = textField.innerHTML;
+    const highlightedContent = searchContent.replace(searchRegex, '<span style="background-color: yellow;">$&</span>');
```

Comment:
HTML has special tag <mark> for highlight, you might want to consider using it. Although I am not sure if it's a best practice but is surely used for the purpose of highlighting.

Reply:
Thank you! I didn't think about using an actual tag for the highlighting. It's unfortunate that the highlight isn't as visible as I would like, though.

Reply:
There is the <mark> tag, which highlights text. However, it is better practice to use CSS to handle styling.

Reply:
Avoid using in line CSS, just add a class instead.